tests/MLNX2700/full_no_hostif_entry.rec

```diff
@@ -49647,3 +49647,10 @@
 2017-05-11.01:45:41.511250|r|SAI_OBJECT_TYPE_NEXT_HOP_GROUP_MEMBER:oid:0x2d000000000656
 2017-05-11.01:44:02.533119|a|APPLY_VIEW
 2017-05-11.01:44:02.535313|A|SAI_STATUS_SUCCESS
+2024-11-02.06:13:50.487652|p|BUFFER_POOL_WATERMARK_STAT_COUNTER|POLL_INTERVAL=60000|BUFFER_POOL_PLUGIN_LIST=78f7658e05f9659b8ecaf00eb0214957a509e246
```

Comment:
Is that plugin list string valid?

Reply:
Yes. It's generated on a physical switch.